### ISSUE TITLE
Add support for output confidence intervals to Workbench

### ISSUE THREAD

**Issue by jakubklimek** [silk-framework/silk#90]
In Silk singlemachine (XML config) one can configure multiple outputs with different confidence intervals. I did not find this setting in Workbench. Is it there? And if not, could this be added? I.e. a linking task could have multiple outputs with confidence intervals specified.

**Comment by skarampatakis** [comment]
You could define multiple linking tasks with different confidence intervals
and diferrent output files.

Στις 8 Μαρ 2017 09:12, ο χρήστης "Jakub Klímek" <notifications@github.com>
έγραψε:

**Comment by jakubklimek** [comment]
True, however, I did not find the confidence interval setting even for a single linking task in Workbench. When it produces links with e.g. 100% confidence and 60% confidence, both kinds are written to the output and the only way to filter the less confident out is to adjust the thresholds and weights in the linking task instead of just simply setting the confidence threshold to 0.7 like you can do in the XML config.

I admit this is rather a convenience issue.
